ConsoleApp1/list.cs

```diff
@@ -0,0 +1,26 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace ConsoleApp1
+{
+    class list
+    {
+        public virtual void Func()
```

Comment:
That is not a cycle that enumerate all figures at one. Read more about Generic collection and polymorphism itself :)

Reply:
Also Func again